content/reference/plugins/github_actions/index.md

```diff
@@ -0,0 +1,189 @@
+
+### Outputs
+
+Example:
+
+```yaml
+steps:
+  - id: gatling-enterprise-action
+    uses: gatling/enterprise-action@v1
+    with:
+      api_token: ${{ secrets.GATLING_ENTERPRISE_API_TOKEN }}
+      simulation_id: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
+  - run: |
+      echo "run_id=${{ steps.enterprise-action.outputs.run_id }}"
+      echo "reports_url=${{ steps.enterprise-action.outputs.reports_url }}"
+      echo "runs_url=${{ steps.enterprise-action.outputs.runs_url }}"
+      echo "runs_status_code=${{ steps.enterprise-action.outputs.runs_status_code }}"
+      echo "runs_status_name=${{ steps.enterprise-action.outputs.runs_status_name }}"
+      echo "run_assertions=${{ steps.enterprise-action.outputs.run_assertions }}"
+```
+
+- `run_id`: The ID of the run started by this action.
+- `reports_url`: The URL of the reports page for this run.
+- `runs_url`: The URL of the runs history page for this simulation.
+- `runs_status_name`: The name of the run's final status (e.g. `Successful`, `AssertionsSuccessful`, `AssertionsFailed`, etc.).
+- `runs_status_code`: The code of the run's final status.
+- `run_assertions`: The results of the run's assertions, as a JSON array.
+
+### Logs
+
+Every few seconds, the action logs to the GitHub Action console a summary of the run's current status. When the run ends, the Action logs the status of the run and the results of any assertions. Here's a very short duration example:
+
+{{< img src="reference_logs.png" alt="A run's logs in the GitHub Actions console" >}}
+
+### Cancellation
+
+When the Action starts, it registers a post-execution, clean-up task in the workflow. If the Action fails or gets cancelled by a user, and if the simulation is still running on Gatling Enterprise, this clean-up task will attempt to cancel the execution on Gatling Enterprise.
+
+## Sample use cases
+
+### Build and run simulation
+
+This workflow is defined in the GitHub repository which contains your Gatling simulation script built with one of our build tools plugins. In this example, every time the code on the `main` branch gets updated, we:
+
+- build, package, and upload to Gatling Enterprise the current version of the simulation script
+- run the updated simulation on Gatling Enterprise 
+
+Feel free to use different trigger events or to configure the other inputs and outputs for the Action as documented above, according to your own use case.
+
+```yaml
+name: Run Gatling Enterprise Simulation
+
+# Execute the workflow on each push to the main branch
+on:
+  push:
+    branches:
+      - main
+
+# Here we use concurrency to cancel previous executions if they are still
+# ongoing. Useful to avoid running the same simulation several times
+# simultaneously if we push several code changes in a short time.
+# See https://docs.github.com/actions/using-jobs/using-concurrency.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
That's apparently a relatively new feature, and it means `styfle/cancel-workflow-action` is no longer required in typical use cases. 🎉